docs/release-notes/3790-login-redirect.txt

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Bug Fixes**
+
+-  Fix an issue where the authentication flow did not remember the originally
+   requested page to redirect to after a successful authentication.
```

Comment:
If you say something like "after a **previously** successful authentication", it would make it a little more clear as to what the context was.